src/opcode_list.h

```diff
@@ -47,3 +47,5 @@ OP(GENLABEL, NONE, 0, 1)
 
 OP(DESTRUCTURE_ALT, BRANCH, 0, 0)
 OP(STOREVN, VARIABLE, 1, 0)
+
+OP(ERROR, CONSTANT, 1, 0)
```

Comment:
Is `1, 0` correct here? I think `stack_in == 1` is correct, but the second value (`stack_out`) is not used for anything so I am not sure what it should be.